### ISSUE TITLE
Plugin doesn't refresh from search

### ISSUE THREAD

**Issue by shmaram** [PagerDuty/backstage-plugin#54]
**Describe the bug**
When searching from the Backstage search modal the data of the plugin doesn't refresh

**To Reproduce**
1. Go to an entity that uses the plugin, you should see the PD data of that service (in the example - "acc")
![image](https://github.com/PagerDuty/backstage-plugin/assets/12017112/9d49f186-619e-4aa5-a031-5c8e785ae444)
2. Click on the "search" from the left pane
![image](https://github.com/PagerDuty/backstage-plugin/assets/12017112/a86e22f4-7ef1-4ab5-b583-78c50e5592a5)
3. You should see the search modal
4. select a different service from the modal data
5. Look at the PD plugin and see that the data didn't change 
![image](https://github.com/PagerDuty/backstage-plugin/assets/12017112/f7ab2f3c-66bf-4012-a5b8-42f73a4a74f3)

**Expected behavior**
After clicking on a different service, the PD data should refresh

**Screenshots**
Added in the steps above

**Desktop (please complete the following information):**
 - OS: Mac Sonoma 14.2.1
 - Browser Chrome
 - Version 120.0.6099.129

**Additional context**
We are connecting the PD plugin using the annotation `pagerduty.com/service-id` and the package is using `EntityPagerDutyCard` from `@pagerduty/backstage-plugin` version: 0.7.3

**Comment by t1agob** [comment]
Thanks for reporting this bug @shmaram. With your detailed reproduction steps I was able to reproduce the bug and will start working on it immediately.

**Comment by t1agob** [comment]
@shmaram just to let you know that I've identified and fixed the issue reported yesterday (PR #55).

I'm working on getting this merged into main and release ASAP. I'll notify you here once it is done.

**Comment by t1agob** [comment]
This issue is now fixed and available on version [0.7.4](https://www.npmjs.com/package/@pagerduty/backstage-plugin). Thanks for reporting it @shmaram!

**Comment by shmaram** [comment]
Just check and it works like a charm!
Thank you @t1agob for the fast response!
